src/App.js

```diff
@@ -1,25 +1,42 @@
-import React from 'react';
 
+import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
+
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    name: 1603367890123,
+  };
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+  handleClick = () => {
+    this.setState(state => ({
+      isClockVisible: !state.isClockVisible,
+    }));
+  }
 
+  randomNameAdd = () => {
```

Comment:
назви функцій краще починати з дієслова
```suggestion
  addRandomName = () => {
```